src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  background-color: black;
```

```suggestion
  background-color: #000;
```
стараемся указывать код